OrbitLinuxTracing/PerfEvent.h

```diff
@@ -185,6 +185,58 @@ struct dynamically_sized_perf_event_stack_sample {
       : stack{dyn_size} {}
 };
 
+struct dynamically_sized_perf_event_call_chain_sample {
```

Comment:
Don't split "callchain" in two words as the `perf_event_open` manpage doesn't. Here and everywhere else.

Reply:
Thanks. I'll rename it.